frontends/p4/specialize.h

```diff
@@ -127,6 +167,21 @@ class Specialize : public Transform {
     const IR::Node* postorder(IR::Declaration_Instance*) override;
 };
 
+/** @brief Repeats ConstantFolding, TypeChecking, FindSpecializations, and
```

Comment:
I used the names of the passes because Doxygen automatically inserts links to their documentation.  But I'm fine replacing that with a more generic description.